client/src/components/molecules/AgendaTag.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Box, Text, Tag } from "@/components/atoms";
+
+interface Tags {
+  hidden: boolean;
+  anonymous: boolean;
+  votable: boolean;
+}
+
+interface Props {
+  tags: Tags;
+}
+
+// TODO: 텍스트 확정해서 알맞게 변경하기
+export const AgendaTag: React.FC<Props> = ({ tags }) => {
+  return (
+    <Box dir="row" gap={8} align="center">
+      <Tag type={tags.hidden ? "hidden" : "_hidden"} />
+      <Tag type={tags.anonymous ? "anonymous" : "_anonymous"} />
+      <Tag type={tags.votable ? "votable" : "empty"} />
```

Comment:
태그를 렌더하고 싶지 않은 것이라면 tag에 `empty` type을 만드는 것보다는 아예 렌더하지 않으면 되지 않을까요?
```suggestion
      {tags.votable && <Tag type="votable" />}
```